docs/installation/building-from-source.rst

```diff
@@ -210,9 +233,10 @@ Many of Pillow's features require external libraries:
 
     Prerequisites are installed on **FreeBSD 10 or 11** with::
 
-        sudo pkg install jpeg-turbo tiff webp lcms2 freetype2 openjpeg harfbuzz fribidi libxcb
+        sudo pkg install jpeg-turbo tiff webp lcms2 freetype2 openjpeg harfbuzz fribidi libxcb libavif
 
-    Then see ``depends/install_raqm_cmake.sh`` to install libraqm.
+    See ``depends/install_raqm_cmake.sh`` to install libraqm and
+    ``depends/install_libavif.sh`` to install libavif.
```

Comment:
As a reader, I find this confusing. It makes me wonder if it I should `pkg install libavif` or `depends/install_libavif.sh`.

Reply:
I've suggested removing the reference to `install_libavif.sh` from this sentence in https://github.com/fdintino/Pillow/pull/6